packages/server/graphql/private/mutations/updateEmail.ts

```diff
@@ -10,19 +10,35 @@ const updateEmail: MutationResolvers['updateEmail'] = async (_source, {oldEmail,
     throw new Error('New email is the same as the old one')
   }
 
-  const user = await getUserByEmail(oldEmail)
+  const user = await pg
+    .selectFrom('User')
+    .selectAll()
+    .where('email', '=', oldEmail)
+    .executeTakeFirst()
+
   if (!user) {
     throw new Error(`User with ${oldEmail} not found`)
   }
 
-  // RESOLUTION
+  //Use identities array for the user above
+  const identitiesArray = user.identities
+
+  // Update the identities array for AuthIdentityLocal, isEmailVerified=false
+  if (identitiesArray && identitiesArray.length > 0) {
+    const localIdentity = (identitiesArray as JsonObject[]).find((identity: JsonObject) => identity.type === 'LOCAL')
```

Comment:
+1 typing in the `find` is not necessary as you've cast already the array itself.

Reply:
Made the required changes to all review comments and updated the PR